.semaphore/semaphore.yml

```diff
@@ -16,4 +16,5 @@ blocks:
             - 'export PATH=/home/semaphore/go/bin:$PATH'
             - checkout
             - go get ./...
-            - go test ./...
+            - go build ./...
+            - go test ./... -v

```

Comment:
build and test should be separate blocks in semaphore.yml. instead of just in command line. the sequence of the blocks in semaphore should be:
1. lint
2. build
3. test

Reply:
that makes sense, I'll make sure to do that in the future. I've captured a JIRA for this: https://confluentinc.atlassian.net/jira/software/projects/METRICS/boards/173?selectedIssue=METRICS-2118